src/routes/routes.js

```diff
@@ -19,5 +19,6 @@ router.get('/api/v1/verify-email/:token', verifyToken, UserAuth.verifyEmail);
 router.post('/api/v1/send-email', Validation.emailValidation, UserAuth.emailSender);
 router.post('/api/v1/reset-password/:token', verifyToken, Validation.passwordValidation, UserAuth.resetPassword);
 router.post('/api/v1/login', Validation.loginValidation, UserAuth.login);
+router.post('/api/v1/signout', [verifyToken], UserAuth.signoutUser);
```

Comment:
I think the route here should be `/api/v1/users/signout.

Reply:
thanks malaba. done implementing it